server/utils/noIndexMiddleware.test.js

```diff
@@ -0,0 +1,50 @@
+const httpMocks = require('node-mocks-http')
+
+const noIndexMiddleware = require('./noIndexMiddleware')
+
+const mockedServerHost = 'mockedserverhost.example.com'
+jest.mock('../configuration', () => ({
+  server: { hostUrl: 'https://mockedserverhost.example.com' },
+}))
+
+describe('noIndexMiddleware', () => {
+  it('should not set "X-Robots-Tag" header if "X-Forwarded-Host" is missing', () => {
+    const next = jest.fn()
+    const { req, res } = httpMocks.createMocks({
+      headers: {},
+    })
+
+    noIndexMiddleware(req, res, next)
+
+    expect(res.getHeader('X-Robots-Tag')).toBeUndefined()
+    expect(next).toHaveBeenCalledTimes(1)
```

Comment:
This is a good solution. The reason why this works in KP, is that we there use `jest-extended`

In [KP's jest.config.js](https://github.com/KTH/kopps-public-react-web/blob/56e9d6f73f99510208c49145c4e35e12ab4ba9df/jest.config.js#L9):
`setupFilesAfterEnv: ['jest-extended/all'],`

If you want, you can also include `jest-extended` in kursinfo-web and include it as above.

Reply:
Thanks for sharing this, I have added the est-extended package and the required config. I also changed "toHaveBeenCalledTimes()" to "toHaveBeenCalledOnce()".